Library/Formula/ipe.rb

```diff
@@ -38,7 +38,7 @@ class Ipe < Formula
 
   def install
     cd 'src' do
-      system "make", "IPEPREFIX=#{prefix}"
+      system "make", "IPEPREFIX=#{HOMEBREW_PREFIX}"
```

Comment:
Yeah, I noticed it after yesterday's exchange. Thanks for doing it, Dominyk. You've effectively solved the big problem. I now simply install CGAL ipelets manually.